supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/request/BatchRequestExecutor.scala

```diff
@@ -188,19 +188,15 @@ object BatchRequestExecutor {
         val time = java.time.Instant
           .ofEpochSecond(e.getEventTime.getSeconds, e.getEventTime.getNanos.toLong)
           .atOffset(java.time.ZoneOffset.UTC)
-        val eventType = e.getDescription match {
-          case startedRegex() => CallMetadataKeys.VmStartTime
-          case endedRegex() => CallMetadataKeys.VmEndTime
-          case _ => e.getType
-        }
-        val executionEvents = List(ExecutionEvent(name = eventType, offsetDateTime = time))
 
-        // Add an additional ExecutionEvent to capture other info if the event is a VmStartTime or VmEndTime
-        if (eventType == CallMetadataKeys.VmStartTime || eventType == CallMetadataKeys.VmEndTime) {
-          executionEvents :+ ExecutionEvent(name = e.getDescription, offsetDateTime = time)
-        } else {
-          executionEvents
-        }
+        // If this event represents the VM starting or stopping, add a special event describing that.
+        val startOrEndEvent = (e.getDescription match {
+          case startedRegex() => Option(CallMetadataKeys.VmStartTime)
+          case endedRegex() => Option(CallMetadataKeys.VmEndTime)
+          case _ => None
+        }).map(t => ExecutionEvent(name = t, offsetDateTime = time))
+
+        List(ExecutionEvent(name = e.getDescription, offsetDateTime = time)) ++ startOrEndEvent.toList
```

Comment:
Now we are consistently using the description when generating events.

Reply:
nice catch!